src/common/favaccordian.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import * as moment from "moment";
+const Favaccordion = ({ children, props, sdata }) => {
+  const [isOpen, setOpen] = useState(false);
+  let d1 = moment(props.data.departureTime);
+  let d2 = moment(sdata.arrivalTime);
+  var startTime = moment(props.data.departureTime, "hh:mm:ss");
+  var endTime = moment(sdata.arrivalTime, "hh:mm:ss");
+  var diff = endTime.diff(startTime, "hours");
+  var totalMinutes = endTime.diff(startTime, "minutes");
+  var clearMinutes = totalMinutes % 60;
+  const addtofavourite = () => {
+    var newArray = JSON.parse(sessionStorage.getItem("fav"));
+    //var newArray = [sessionStorage.getItem("fav")];
+    newArray = newArray ? newArray : [];
+    //console.log(newArray);
+    newArray = [...newArray, props];
+    sessionStorage.setItem("fav", JSON.stringify(newArray));
+    console.log(JSON.parse(sessionStorage.getItem("fav")));
+  };
+  const clear = () => {
+    sessionStorage.clear();
+  };
+  if (diff > 0) {
```

Comment:
use conditional operator to render content . we do not use if else for rendering.